develdlang.Dockerfile

```diff
@@ -32,4 +32,4 @@ LABEL \
     com.sociomantic.version.hmod=$VERSION_HMOD
 
 COPY docker/ /docker-tmp
-RUN cd /docker-tmp && ./dlang && rm -fr /docker-tmp
+RUN cd /docker-tmp && ./develdlang && rm -fr /docker-tmp
```

Comment:
Will there ever be non-devel `dlang` image?

Reply:
I was going for consistency, to indicate it's based on the `develbase`, but apparently the new, consistent, names are a bit controversial.